pkg/daemon/ceph/client/pool.go

```diff
@@ -327,7 +327,7 @@ func setCommonPoolProperties(context *clusterd.Context, clusterInfo *ClusterInfo
 	}
 
 	if pool.Replicated.IsTargetRatioEnabled() {
-		pool.Parameters[targetSizeRatioProperty] = strconv.FormatFloat(pool.Replicated.TargetSizeRatio, 'f', -1, 32)
+		pool.Parameters[targetSizeRatioProperty] = strconv.FormatFloat(*pool.Replicated.TargetSizeRatio, 'f', -1, 32)
 	}
 
 	if pool.IsCompressionEnabled() {
```

Comment:
Similar change for the compression mode is needed
```suggestion
	if _, ok := pool.Parameters[CompressionModeProperty]; !ok {
```